src/cli.rs

```diff
@@ -94,6 +94,9 @@ pub enum Command {
 
     #[structopt(setting(structopt::clap::AppSettings::Hidden))]
     Install(command::Install),
+
+    /// Show info about your environment for help debugging
+    Info(command::Info),
```

Comment:
we should probably have this be a hidden command and then in our issue tracker let people know they can run it. could also run it automatically on the error types that say "submit an issue" and could also include it in `robot_panic` 😄

Reply:
Ooo those are great recommandations!